tests/app/forms/test_fields.py

```diff
@@ -304,12 +304,61 @@ def test_checkbox_field(self):
         unbound_field = get_field(checkbox_json, checkbox_json['label'], error_messages, self.answer_store,
                                   self.metadata)
 
-        expected_choices = [(option['label'], option['value']) for option in checkbox_json['options']]
+        expected_choices = [(option['value'], option['label']) for option in checkbox_json['options']]
 
         self.assertTrue(unbound_field.field_class == SelectMultipleField)
         self.assertEqual(unbound_field.kwargs['label'], checkbox_json['label'])
         self.assertEqual(unbound_field.kwargs['description'], checkbox_json['guidance'])
         self.assertEqual(unbound_field.kwargs['choices'], expected_choices)
+        self.assertEqual(len(unbound_field.kwargs['validators']), 1)
+
+
+    def test_mutually_exclusive_checkbox_field(self):
+        checkbox_json = {
+            'guidance': '',
+            'id': 'opening-crawler-answer',
+            'label': '',
+            'mandatory': False,
+            'options': [
+                {
+                    'label': 'Luke Skywalker',
+                    'value': 'Luke Skywalker'
+                },
+                {
+                    'label': 'Han Solo',
+                    'value': 'Han Solo'
+                },
+                {
+                    'label': 'The Emperor',
+                    'value': 'The Emperor'
+                },
+                {
+                    'label': 'R2D2',
+                    'value': 'R2D2'
+                },
+                {
+                    'label': 'Senator Amidala',
+                    'value': 'Senator Amidala'
+                },
+                {
+                    'label': 'I prefer star trek',
+                    'value': 'None'
+                }
+            ],
+            'type': 'MutuallyExclusiveCheckbox'
+        }
+
+        unbound_field = get_field(checkbox_json, checkbox_json['label'], error_messages, self.answer_store,
+                                  self.metadata)
+
+        expected_choices = [(option['value'], option['label']) for option in checkbox_json['options']]
+
+        self.assertTrue(unbound_field.field_class == SelectMultipleField)
```

Comment:
Why not `assertEqual`?

Reply:
That was a copy and paste without thinking. Changed in the original as well.